platform/broadcom/saibcm-modules/debian/changelog

```diff
@@ -1,3 +1,9 @@
+opennsl (6.1.0.3) unstable; urgency=medium
```

Comment:
Can you do a similar change in https://github.com/Azure/saibcm-modules/commits/sdk-6.5.24-gpl-dnx.
Also update the submodule in platform/broadcom/saibcm-modules-dnx

Reply:
Done